### ISSUE TITLE
Dungeon Doors Crashing? [WiiU VC]

### ISSUE THREAD

**Issue by Sparky7198** [OoTRandomizer/OoT-Randomizer#308]
Okay so i was testing the Latest Dev Build (3.5.4) of Ocarina of time Randomizer on a Wii u Console Using Loadiine (had to inject it through an official Ocarina of time Virtual Console Rom) And for some Reason The Doors in Deku Tree Were Crashing when i tried to enter them. it proved to be consistant but then i saved and reloaded my file in the Dungeon and the doors now work but they don't hide my hud or Change the Camera anymore. like the doors on the Chest Minigame and when i re-entered the dungeon without saving they Crashed again. i even tried Deleting the VC's 1.2 patches but it still crashed,Using a Compressed Rom too,not sure if i have to leave it uncompressed

**Comment by flagrama** [comment]
So while I'm sure the devs will want to try to get this all to work with the Wii U VC at some point too, have you tried just installing a Wii WAD of the rando to your vWii? It is definitely known to work with the Wii VC already.

**Comment by Sparky7198** [comment]
I don't use VirtualWii on my console,heck i barely even used it.

**Comment by DiegoM390** [comment]
It appears I've seen to have fixed the door issue with the Randomizer. Testing newer versions of the Randomizer (specially randomized dungeons) I found out the issue was present in every dungeon in the game using OOT as the base. I heard not too long ago that an OOT romhack was able to be completed 100% by using Majora's mask as a Base so I went and tried it for myself with the Randomizer... And so far at least in the water temple, doors seems to be behaving like they should be and are not crashing the game.
